raygun4go.go

```diff
@@ -191,14 +191,14 @@ func (c *Client) CreateError(message string) error {
 func (c *Client) SendError(error error) error {
 	err := errors.New(error.Error())
 
-  var st stackTrace = nil
-  if goerror, ok := error.(*goerrors.Error); ok {
-	  st = make(stackTrace, 0, 0)
-	  stack2struct.Parse(goerror.Stack(), &st)
-  } else {
-  	st = currentStack()
-  }
-	
+	var st stackTrace = nil
+	if goerror, ok := error.(*goerrors.Error); ok {
+		st = make(stackTrace, 0, 0)
+		stack2struct.Parse(goerror.Stack(), &st)
+	} else {
+		st = currentStack()
+	}
+
 	post := c.createPost(err, st)
```

Comment:
Sorry about these changes on format, my vim is configured to use `go fmt`. Let me know if you prefer to merge only the significant changes.